spec/jobs/sync_case_comments_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe Jobs::SyncCaseComments do
+  include_context "spec helper"
+
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:_case) { Fabricate(:salesforce_case, topic: topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+  fab!(:post2) { Fabricate(:post, topic: topic, post_number: 2) }
+  fab!(:post3) { Fabricate(:post, topic: topic, post_number: 3) }
+
+  it 'creates multiple case comment objects on Salesforce' do
+    topic.custom_fields["has_salesforce_case"] = true
+    topic.save!
+
+    ::Salesforce::CaseComment.any_instance.expects(:create!).twice
```

Comment:
Same concern as https://github.com/discourse/discourse-salesforce/pull/4/files#r823371345

Reply:
Same as the https://github.com/discourse/discourse-salesforce/pull/4#discussion_r824278941